sdk/aqueduct/utils/utils.py

```diff
@@ -105,7 +105,12 @@ def parse_user_supplied_id(id: Union[str, uuid.UUID]) -> str:
 
 
 def construct_param_spec(val: Any, artifact_type: ArtifactType) -> ParamSpec:
-    serialization_type = artifact_type_to_serialization_type(artifact_type, val)
+    serialization_type = artifact_type_to_serialization_type(
+        artifact_type,
+        # not derived from bson. This is irrelevant for now as we don't support table parameters.
```

Comment:
We support parameters of any type, so I do think we support table parameters; we just don't expect the table parameter to be serialized in the bson form, so I think this change is still fine. If so we just need to update this comment to be more accurate.